webapp/components/wpt-results.js

```diff
@@ -534,12 +528,10 @@ class WPTResults extends WPTColors(WPTFlags(SelfNavigation(LoadingState(TestRuns
     if (!this.showTestRefURL || testType !== 'reftest') {
       return;
     }
-    const item = Object.values(manifest.items['reftest']).find(v => v.find(i => i[0] === path));
-    // In item[0], the 2nd item is the refs array, and we take the first ref (0).
-    // Then, the ref's 1st item is the url (0). (2nd is the condition, e.g. "==".)
-    // See https://github.com/web-platform-tests/wpt/blob/master/tools/manifest/item.py#L141
-    const refPath = item && item[0][1][0][0];
-    return this.computeTestURL(testType, refPath);
+    const metadata = manifest.get(path);
+    if (metadata && metadata.refPath) {
+      return this.computeTestW3CURL(testType, metadata.refPath);
```

Comment:
I don't see `computeTestW3CURL` anywhere in master or this patch? Where's it defined?